presto-main/src/main/java/com/facebook/presto/connector/informationSchema/InformationSchemaPageSourceProvider.java

```diff
@@ -202,6 +208,34 @@ private InternalTable buildTables(Session session, String catalogName, Map<Strin
         return table.build();
     }
 
+    private InternalTable buildTablePrivileges(Session session, String catalogName, Map<String, NullableValue> filters)
+    {
+        List<GrantInfo> grants = ImmutableList.copyOf(getTablePrivilegesList(session, catalogName, filters));
+
+        InternalTable.Builder table = InternalTable.builder(informationSchemaTableColumns(TABLE_TABLE_PRIVILEGES));
+        for (GrantInfo grant : grants) {
+            if (session.getIdentity().getUser().equals(grant.getIdentity().getUser())) {
```

Comment:
Good point. I will add it once we add information about the grantor (not exactly here though, since I changed this piece of code).